pyspark/test/bigdl/keras1/test_layer.py

```diff
@@ -155,14 +218,18 @@ def test_globalaveragepooling1d(self):
     def test_batchnormalization(self):
         input_data = np.random.random_sample([2, 6, 128, 128])
         layer = BatchNormalization(input_shape=(6, 128, 128), axis=1)
-        self.modelTestSingleLayer(input_data, layer,
-                                  dump_weights=True)
+        self.modelTestSingleLayer(input_data, layer, dump_weights=True, random_weights=False)
 
     def test_flatten(self):
         input_data = np.random.random_sample([1, 2, 3])
         layer = Flatten(input_shape=(2, 3))
         self.modelTestSingleLayer(input_data, layer)
 
+    def test_permute(self):
+        input_data = np.random.random_sample([5, 4, 3, 2, 6])
+        layer = Permute((4, 1, 2, 3), input_shape=(4, 3, 2, 6))
```

Comment:
add more test here. i.e 4, 3, 2, 1;  1, 2, 3, 4;  4, 4, 2, 3

Reply:
OK

Reply:
4, 4, 2, 3 ??
This is not a valid permutation... maybe you mean 1, 4, 2, 3?